SocialFilter/sections/general_elemhide.txt

```diff
@@ -972,6 +972,7 @@ designboom.com,livescience.com,pbtech.co.nz,peoplesworld.org,space.com##.fa-face
 ##.social__likes
 ##.social_bar
 ##.social_btn
+##.social__btn
```

Comment:
Please make a specific rule, there is no sense for making a general rule.
`finalfantasyxiv.com##.l__blogsocial`